mavis/python/mavis_tacplus_vault.py

```
@@ -0,0 +1,591 @@
+					file=sys.stderr,
+				)
+				continue
+			sanitized.append(g)
+		if sanitized:
+			D.set_tacmember('"' + '","'.join(sanitized) + '"')
+
+		# Cache sanitized groups so INFO/HOST reads don't need re-sanitization
+		_cache_put(D.user, sanitized, D.user)
+		D.write(MAVIS_FINAL, AV_V_RESULT_OK, None)
+
+	except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)